docs/reference/api/pytorch.txt

```diff
@@ -170,6 +170,18 @@ your code:
    to the ``clip_grads`` argument of ``step_optimizer``
    while removing the ``PyTorchCallback`` counterpart in the :meth:`build_callbacks` method.
 
+Debugging FAQ
```

Comment:
Move this to a pytorch section of the main FAQ.

Reply:
We don't have the main FAQ and I don't think putting everything in one main FAQ would be helpful to users. Basically we haven't discussed how we would have FAQ. I think it's fine to just put that here.

Reply:
Yes we do: https://docs.determined.ai/latest/faq.html, and it already has a "tensorflow support" section.

Reply:
oops. I am surprised that was not linked to the framework-specific doc. Moved the doc to that main FAQ.